### PR TITLE
next app directory routing

### PR DESCRIPTION
**Pull request by brycedev**

### CODE REVIEW

**Comment by PeerRich** [comment]
once this is merged i'll add /issues and /pull-requests pages

**Comment by PeerRich** [comment]
CleanShot 2023-03-06 at 00 08 24@2x

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true
+}
```
Comment (pierresisson) [inline .vscode/settings.json:4]: don't push vscode settings ?

File: next.config.mjs [hunk 2]
```diff
@@ -16,9 +19,9 @@ const config = {
...
    * @see https://github.com/vercel/next.js/issues/41980
    */
-  i18n: {
-    locales: ["en"],
-    defaultLocale: "en",
-  },
+  // i18n: {
+  //   locales: ["en"],
+  //   defaultLocale: "en",
+  // },
```
Comment (pierresisson) [inline next.config.mjs:25]: remove comments ?

File: src/app/layout.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        style={{ minWidth: 1080 }}
+        className="dark min-h-screen bg-white text-white dark:bg-[#0E1116]"
+      >
+          <Header />
+          {children}
+        </div>
+      </body>
+    </html>
+  );
+}
```
Comment (pierresisson) [inline src/app/layout.tsx:22]: prettier

File: src/app/page.tsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import { redirect } from 'next/navigation'
+
+export default function IndexRedirect({  }) {
+  redirect('/nostr-protocol/nostr')
+}
```
Comment (pierresisson) [inline src/app/page.tsx:5]: prettier

**Review by pierresisson — CHANGES_REQUESTED** [review]

**Comment by pierresisson** [comment]
@PeerRich we need to merge this until doing other stuffs

File: .vscode/settings.json (hunk 1 above)
Comment (PeerRich) [reply .vscode/settings.json:4]: agree, lets add this to .gitignore too

File: next.config.mjs (hunk 2 above)
Comment (brycedev) [reply next.config.mjs:25]: i kept this one specifically because of the next-included comment, but generally agree

Comment (PeerRich) [inline next.config.mjs:25]:
```suggestion
```

**Review by PeerRich — APPROVED** [review]

File: src/app/page.tsx (hunk 4 above)
Comment (PeerRich) [reply src/app/page.tsx:5]: nice

**Comment by PeerRich** [comment]
@pierresisson is this safe to merge? we can address NITs in a follow up

**Pull request merged**